Index: sandbox/src/main/java/MyFirstProgram.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- sandbox/src/main/java/MyFirstProgram.java	(revision c15ae5e9eb2aa408dbcc864ff8c2399f5fb60718)
+++ sandbox/src/main/java/MyFirstProgram.java	(revision )
@@ -1,8 +1,5 @@
 public class MyFirstProgram {
 
- public static void main(String[] args) {
-	System.out.println("Hello, world!");
-
-  }
+ public static void main(String[] args) { System.out.println("Hello, world!"); }
 
 }
\ No newline at end of file
